en/health/centreon-agent.md

```diff
@@ -0,0 +1,473 @@
+---
+id: centreon-agent
+title: Installing the Centreon Agent
+---
+
+## Overview
+
+The Centreon Agent is a light piece of software that monitors its host machine and the services that run on it.
+
+The Agent is used to monitor servers that operate an On-Premise Centreon service (Central, Remote Server, Poller, Map, etc.).
+
+The data is sent to the Centreon Cloud Platform. No personal data is collected.
+
+>Although the following procedure and the Agent configuration files in general allow for some customization, we strongly advise you to leave the filenames etc. as shown here.
+
+## Requirements
+
+- In order for the metrics to reach the Centreon Cloud Platform (where the monitoring of the monitoring is done), a Centreon Agent must be able to access our public endpoint at the following URL:
+
+    ```https://api.a.prod.mycentreon.com/v1/observability``` (port 443)
+
+    You can test whether your machine can access our endpoint using the following command:
+
+    ```
+    curl -v https://api.a.prod.mycentreon.com/v1/observability
+    ```
+
+    You can also go through a proxy using the following command:
+
+    ```
+    curl -v https://api.a.prod.mycentreon.com/v1/observability \
+    --proxy [protocol://]host[:port] --proxy-insecure
+    ```
+
+    Example:
+
+    ```
+    curl -v https://api.a.prod.mycentreon.com/v1/observability \
+    --proxy http://proxy.local.net:3128 --proxy-insecure
+    ```
+
+    The following message will be returned in case of success:
+
+    ```
+    "Missing Authentication Token"
+    ```
+
+    If you receive a different answer or no answer, your machine cannot reach our endpoint, likely due to your network rules (firewall, proxy, etc.).
+
+    >If a proxy access is configured on the host machine, you need to copy the address and port of the proxy to the Agent’s configuration file (see section [Network](#network)).
+
+- If a host machine does not have direct access to the outside, two options that complement each other are provided: [proxy configuration](#proxy-configuration) and [gateway configuration](#gateway-configuration).
+
+- The RPMs are available in the Centreon official repositories for the currently supported versions. The official Centreon repository must be installed:
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--RHEL / CentOS / Oracle Linux 8-->
+```shell
+dnf install -y http://yum.centreon.com/standard/21.04/el8/stable/noarch/RPMS/centreon-release-21.04-4.el8.noarch.rpm
+```
+<!--CentOS 7-->
+```shell
+yum install -y http://yum.centreon.com/standard/21.04/el7/stable/noarch/RPMS/centreon-release-21.04-4.el7.centos.noarch.rpm
+```
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+- You must be in possession of your unique token that allows you send data to our platform. This token is provided to you by our Support team. If you are using a [gateway server/client](#network) system, you need a separate token for the gateway.
```

Comment:
I am no so sure about how this is presented:

- Using a token for gateway security is completely optional and has nothing to do with the authentication token
- This is also misleading for it can be read at "ask a different Token for that" where this is smth entirely in the hands of the user
- This is an advance security feature so I would not recommend putting it front and center